src/Pane.tsx

```diff
@@ -46,8 +47,26 @@ export const InnerPane = ({
   bgColor,
   borderWidth,
   borderColor,
-  children
+  children: _children
 }: PropsWithChildren<InnerPaneProps>) => {
+  const children = useMemo(() => {
+    if (!_children) return _children;
+    if (typeof _children === 'string') return _children;
+    if (typeof _children === 'number') return _children;
+    if (typeof _children === 'boolean') return _children;
+    if (isIterableReactNode(_children)) return _children;
+
+    // TODO: Inference displayName from the component
+    if ((_children.type as any).displayName === 'PaneSystem') {
+      return createElement(InnerPaneSystem, {
+        ...(_children.props || {}),
```

Comment:
Let's use `??` instead.

Reply:
I thought I had to maintain browser compatibility, but TypeScript transpiles it! Thank you!